purchase_packaging/models/purchase_order_line.py

```diff
@@ -34,7 +35,8 @@ def _default_product_purchase_uom_id(self):
     product_qty = fields.Float(
         compute="_compute_product_qty",
         string='Quantity',
-        inverse='_inverse_product_qty'
+        inverse='_inverse_product_qty',
+        store=True
```

Comment:
you can't do that: this fields value depends on the context (you can compute the quantity in the context of a warehouse, or of a stock location...). Changing product_qty to a stored field will break odoo.